include/grpcpp/security/tls_credentials_options.h

```diff
@@ -55,7 +55,8 @@ class TlsKeyMaterialsConfig {
   }
   int version() const { return version_; }
 
-  /** Setter for key materials that will be called by the user. **/
+  /** Setter for key materials that will be called by the user. They do not take
+   *  ownership of the arguments. **/
```

Comment:
Maybe rephrase it to "ownership of the arguments will not be transferred". It's unclear if what `they` refers to.

Reply:
Done.